extensions/big-data-cluster/src/bigDataCluster/tree/controllerTreeDataProvider.ts

```diff
@@ -69,9 +69,9 @@ export class ControllerTreeDataProvider implements vscode.TreeDataProvider<TreeN
 	}
 
 	public async loadSavedControllers(): Promise<void> {
-		let config = vscode.workspace.getConfiguration(ConfigNamespace);
-		if (config && config.controllers) {
-			let controllers = config.controllers;
+		let controllers: { url: string, username: string, rememberPassword: boolean }[]
```

Comment:
I'd suggest creating a typedef on this so that changes in this or the save code aren't missing the other one and breaking (which wouldn't be noticed until run-time)